tasks/node-setup.yml

```diff
@@ -2,5 +2,6 @@
 - name: Join node to Kubernetes master
   shell: >
     {{ kubernetes_join_command }}
+    --ignore-preflight-errors={{ kubernetes_ignore_preflight_errors }}
```

Comment:
In addition to this, we would need to split the `kubernetes_ignore_preflight_errors ` var in README out into it's own little section, with a description like 'this is used for the `kubeadm init` command as well as the `kubeadm join` command's `--ignore-preflight-errors` option' (see https://github.com/geerlingguy/ansible-role-kubernetes/blob/master/README.md#role-variables)

Reply:
The README has been updated.